src/main/java/bio/terra/cli/command/group/List.java

```diff
@@ -24,8 +29,50 @@ protected void execute() {
 
   /** Print this command's output in text format. */
   private static void printText(java.util.List<UFGroup> returnValue) {
-    for (UFGroup group : returnValue) {
-      group.print();
+    TablePrinter<UFGroup> printer = bio.terra.cli.command.group.List.UFGroupColumns::values;
+    OUT.println(printer.print(returnValue));
+  }
+
+  /** Column information for fields in `resource list` output */
+  private enum UFGroupColumns implements ColumnDefinition<UFGroup> {
+    EMAIL("EMAIL", g -> g.email, 20, LEFT),
```

Comment:
It would be nice if email weren't cut off. Also, members and policies don't need to be so wide. Can you try:

email: 40
members: 9
policies: 10

Can you try this out in verily-devel and verily-prod, to make sure emails aren't cut off?